README.md

```diff
@@ -81,6 +81,20 @@ Then either:
  - Run or Debug the `verification`, `test` Task from the "Gradle" panel
  - Open an individual test class and click the green play icon in the gutter at the class level or on a specific test method
 
+## Running Performance Tests
+
+To run the performance tests use gatlingRun gradle task
```

Comment:
Having something in the readme is a good idea however I think it could be a pointer to the [existing how-to doc](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/doc/how-to/run_performance_tests.md) instead to avoid the duplication and keep the readme concise. 

We should add those instructions for running a specific too, that's very helpful.

Reply:
cool, i wasn't aware of the how-to docs. i've removed this and updated the how-to